src/typing/flow_js.ml

```diff
@@ -3687,6 +3729,7 @@ and ground_subtype = function
   | (VoidT _, UseT (_, VoidT _))
   | (EmptyT _, _)
   | (_, UseT (_, MixedT _))
+  | (_, UseT (_, ClassT (MixedT _)))
```

Comment:
I think that this is part of the PR's "enforce subtyping on structural class statics" clause. It should probably be exported and rebased below this PR.